tcli/tcli_test.py

```diff
@@ -241,18 +241,18 @@ def testStartUp(self):
   def testTildeCompleter(self):
 
     self.assertEqual(
-        'record', self.tcli_obj._TildeCompleter(['reco'], 0))
+        '/record', self.tcli_obj._TildeCompleter('/reco', 0))
```

Comment:
By dropping '/' as a word separator, we now include the TILDE in the command matching.